helm/reana/Chart.yaml

```diff
@@ -28,11 +28,11 @@ keywords:
 type: application
 # Chart version.
 version: 0.8.1
-kubeVersion: ">= 1.13.0-0 < 1.22.0-0"
+kubeVersion: ">= 1.19.0-0 < 1.22.0-0"
 dependencies:
   - name: traefik
-    version: 1.85.x
-    repository: https://charts.helm.sh/stable
+    version: 10.3.4
```

Comment:
Please update `CHANGES.rst` about adding support for Kubernetes 1.22 and 1.23 and about dropping support for versions older than 1.19.  This definitely needs a very visible entry in the change log. As we shall target REANA 0.9 release series for this change, you can directly state `REANA 0.9.0 (UNRELEASED)` in `CHANGES.rst`.

Reply:
So our next version is 0.9.0? We are not having any patch versions?

Reply:
0.8.2 is for bug fixes only for the past stable release series which is 0.8. 

0.9.0 is to start new release series, new features, etc. This nicely fits the deprecation of older Kubernetes versions.